### DISCUSSION TITLE
Set environment variable with same value but different name

### DISCUSSION THREAD

**Discussion by dixiedream** [BretFisher/ama#167] [Q&A]
Hi Bret!

It's possible to set 2 env var with the same value but different name?

example:
```
MONGO_CONNECTION=mongodb://db:27017/test
DB_URI=${MONGO_CONNECTION}
```

Thanks!

By the way! Your docker nodejs course is AWESOME!

**Comment by BretFisher** [comment]
👋 
Thanks for the thanks!

Where are  you talking about putting these ENV's? A Dockerfile, docker-compose file, Swarm Stack, or?

**Comment by dixiedream** [comment]
> Where are you talking about putting these ENV's? A Dockerfile, docker-compose file, Swarm Stack, or?

My typical use case is to place the vars in a .env file for local development then use those with env_file property in the compose file and in a production environment using Swarm secrets.

**Comment by BretFisher** [comment]
Only docker-compose cli supports .env files. You'd need to export those manually to shell before Swarm stack deploy would pick them up.
